src/styles.js

```diff
@@ -9,10 +51,30 @@ export const GlobalStyle = createGlobalStyle`
 
 	html {
 		font-size: 16px;
+
 	}
 
 	body {
 		margin: 0;
+		padding: 50px 100px 50px 100px;
```

Comment:
You can use the shorthand for that if you like :) 

```suggestion
		padding: 50px 100px;
```